frontend/src/lib/components/PropertyFilters/PropertyFilters.js

```diff
@@ -6,8 +6,10 @@ import { propertyFilterLogic } from './propertyFilterLogic'
 import { Popover, Row } from 'antd'
 import { CloseButton } from '../../utils'
 
+const operatorEntries = Object.entries(operatorMap).reverse()
```

Comment:
reversed entries because icontains needs to be checked after not_icontains since they share key. placed in variable here so it's not recalculated in loop everytime